go/vt/mysqlctl/builtinbackupengine.go

```diff
@@ -322,17 +323,22 @@ func (be *BuiltinBackupEngine) executeFullBackup(ctx context.Context, params Bac
 	}
 
 	// get the read-only flag
-	readOnly, err = params.Mysqld.IsReadOnly()
+	readOnly, err := params.Mysqld.IsReadOnly()
```

Comment:
printing read_only for debugging purpose.

Reply:
Is it even necessary to get / check / print readOnly any more? We could delete these lines of code, we are only interested in super_read_only from now on.

Reply:
I want to keep it for debugging purpose initially. This will help me debug if there are production issues. I will remove them later as part of another work item. 
https://github.com/vitessio/vitess/issues/12143